scripts/ensure-pinned-deps.ts

```diff
@@ -21,7 +21,7 @@ const allDeps = {...packageJson.dependencies, ...packageJson.devDependencies};
 const invalidDeps = new Map<string, string>();
 
 for (const [depKey, depValue] of Object.entries(allDeps)) {
-  if (/[0-9]/.test(depValue[0]!)) {
+  if (/[0-9\*]/.test(depValue[0]!)) {
```

Comment:
This kinda defeats the purpose of this whole “ensure pinned deps” script :)

Instead of allowing `*` generally, we could hardcode an exception for the `@pptr/testserver` package, which is the only one you’d like to use `*` for IIUC. WDYT?

Reply:
Done.